### PR TITLE
PARQUET-2064: Make Range public accessible in RowRanges

### PR DESCRIPTION
**Pull request by shangxinli**
Make sure you have checked _all_ steps below.

### Jira

- [ ] My PR addresses the following [Parquet Jira](https://issues.apache.org/jira/browse/PARQUET/) issues and references them in the PR title. For example, "PARQUET-1234: My Parquet PR"
  - https://issues.apache.org/jira/browse/PARQUET-XXX
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Tests

- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [ ] My commits all reference Jira issues in their subject lines. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain Javadoc that explain what it does

### CODE REVIEW

File: parquet-column/src/main/java/org/apache/parquet/internal/filter2/columnindex/RowRanges.java [hunk 1]
```diff
@@ -63,8 +64,8 @@ private static Range intersection(Range left, Range right) {
       return null;
     }
 
-    final long from;
-    final long to;
+    public final long from;
+    public long to;
```
Comment (gszadovszky) [inline parquet-column/src/main/java/org/apache/parquet/internal/filter2/columnindex/RowRanges.java:68]: `to` should also be final.

File: parquet-column/src/main/java/org/apache/parquet/internal/filter2/columnindex/RowRanges.java [hunk 2]
```diff
@@ -299,6 +300,10 @@ public boolean isOverlapping(long from, long to) {
         (r1, r2) -> r1.isBefore(r2) ? -1 : r1.isAfter(r2) ? 1 : 0) >= 0;
   }
 
+  public List<Range> getRanges() {
+    return ranges;
```
Comment (gszadovszky) [inline parquet-column/src/main/java/org/apache/parquet/internal/filter2/columnindex/RowRanges.java:304]: I don't like returning `ranges` directly. This is a mutable list so the caller can modify it without any control. I would suggest wrapping it into an unmodifiable list.

**Review by gszadovszky — CHANGES_REQUESTED** [review]

File: parquet-column/src/main/java/org/apache/parquet/internal/filter2/columnindex/RowRanges.java (hunk 1 above)
Comment (shangxinli) [reply parquet-column/src/main/java/org/apache/parquet/internal/filter2/columnindex/RowRanges.java:68]: Fixed it.

File: parquet-column/src/main/java/org/apache/parquet/internal/filter2/columnindex/RowRanges.java (hunk 2 above)
Comment (shangxinli) [reply parquet-column/src/main/java/org/apache/parquet/internal/filter2/columnindex/RowRanges.java]: Yeah, that makes sense. After this PR, I will change the master branch also.

**Review by gszadovszky — APPROVED** [review]

**Pull request merged**